text_extensions_for_pandas/array/token_span.py

```diff
@@ -764,12 +764,6 @@ def as_frame(self) -> pd.DataFrame:
     ##########################################
     # Keep private and protected methods here.
 
-    def _repr_html_(self) -> str:
```

Comment:
why is this being removed?

Reply:
other than this it looks good to me!

Reply:
That function overrides a function by the same name, with the same function body, in the parent class `CharSpanArray`. I discovered the duplicate functions when I modified `CharSpanArray._repr_html()`. I think the duplicate functions are a relic of the time before `TokenSpanArray` was a subclass of `CharSpanArray`.